caffe2/operators/pool_op.cc

```diff
@@ -900,22 +536,20 @@ std::function<void(OpSchema&)> MaxPoolDocGenerator(const char* dim) {
         0,
         "X",
         "*(type: Tensor`<float>`)* Input data tensor of shape NCHW or NHWC.");
-    schema.Output(
-        0,
-        "Y",
-        "*(type: Tensor`<float>`)* Output data tensor.");
+    schema.Output(0, "Y", "*(type: Tensor`<float>`)* Output data tensor.");
     /*
-    schema.Arg("kernel", "*(type: int)* Size of the window to take an average over.");
-    schema.Arg("stride", "*(type: int)* Stride of the window.");
-    schema.Arg("pad", "*(type: int)* Implicit zero padding to be added on both sides.");
-    schema.Arg("dilation", "*(type: int)* Parameter that controls the stride of elements in the window.");
-    schema.Arg("order", "*(type: string; default: 'NCHW')* Order of the blob dimensions.");
+    schema.Arg("kernel", "*(type: int)* Size of the window to take an average
+    over."); schema.Arg("stride", "*(type: int)* Stride of the window.");
+    schema.Arg("pad", "*(type: int)* Implicit zero padding to be added on both
+    sides."); schema.Arg("dilation", "*(type: int)* Parameter that controls
+    the stride of elements in the window."); schema.Arg("order", "*(type:
+    string; default: 'NCHW')* Order of the blob dimensions.");
     */
   };
 }
 REGISTER_CPU_OPERATOR(
     AveragePool,
-    PoolOp<float, CPUContext, AveragePool<float>>);
+    PoolOp<float, CPUContext, AveragePoolFunctor<CPUContext>>);
 
 OPERATOR_SCHEMA(AveragePool)
```

Comment:
Shall we also add the description about arg `count_include_pad` to the AveragePoolDocGenerator?

Reply:
Actually I separate this approach into two parts. This is the first part to add it into CPU. And then I will add it into GPU part, which is more important. So I'd suggest add the doc when the GPU part finished.